src/components/WelcomeIntro.js

```diff
@@ -1,13 +1,60 @@
 import React from "react";
-import { Col, Container, Row } from "react-bootstrap";
+import { Col, Container, Row, Button } from "react-bootstrap";
+import CocoaBean from "../assets/CocoaBean.svg";
+import JellyBean from "../assets/JellyBean.svg";
+import GardenBean from "../assets/GardenBean.svg";
+import SoyBean from "../assets/SoyBean.svg";
+import MagicBean from "../assets/MagicBean.svg";
+import CabinCupLogo from "../assets/CabinCupLogo.svg";
+
+
+import '../styles/cabincup.scss';
```

Comment:
Could you move this import in `main.scss` just to match what we've been doing elsewhere?